src/index.html

```diff
@@ -6,13 +6,71 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="Product"
+      />
+
+      <section class="card__info">
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <span class="card__code">Product code: 195434</span>
+
+        <div class="card__reviews">
+          <div class="stars stars--rating-4">
+            <div class="stars__star stars__star--active"></div>
+
+            <div class="stars__star stars__star--active"></div>
+
+            <div class="stars__star stars__star--active"></div>
+
+            <div class="stars__star stars__star--active"></div>
+
+            <div class="stars__star"></div>
```

Comment:
It would be better to generate these stars dynamically using a loop in your CSS or JavaScript, rather than hard-coding them. This would make your code more flexible and easier to maintain.